omnibus/config/software/python2.rb

```diff
@@ -25,6 +25,7 @@
   dependency "openssl"
   dependency "bzip2"
   dependency "libsqlite3"
+  dependency "libyaml"
```

Comment:
nit: to me it'd make more sense to define this as a dependency of `six` and `datadog-agent-integrations`

(not important for 6.12)